src/main/java/meditracker/ui/Ui.java

```diff
@@ -109,6 +111,50 @@ public static <T> void printMedsList(List<T> medications) {
         }
     }
 
+    /**
+     * Prints the sub lists of dailyMedications based on the period
+     *
+     * @param medsList list of medications from MedicationManager
+     * @param dailyMedications subList of dailyMedication
+     * @param period Specified period of the day
+     */
+    public static void printMedsLists(List<Medication> medsList,
+                                      List<DailyMedication> dailyMedications, String period) {
+        int numbering = 0;
+        for (DailyMedication med: dailyMedications) {
+            String name = med.getName();
+            Double intakeDose;
+            int index = getIndex(medsList, name);
+            if (period.equals("Morning:")) {
+                intakeDose = medsList.get(0).getDosageMorning();
+            } else if (period.equals("Afternoon:")) {
+                intakeDose = medsList.get(0).getDosageAfternoon();
+            } else {
+                intakeDose = medsList.get(index).getDosageEvening();
```

Comment:
Is this supposed to .get(0) only or .get(index)?

Reply:
Good catch on my mistake, updated to use "index" in c7699f5